ui/src/app/preferences/preferencesSlice.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { createSlice } from '@reduxjs/toolkit';
+import { RootState } from '~/store';
+import { Theme, Timezone } from '~/types/Preferences';
+
+interface IPreferencesState {
+  theme: Theme;
+  timezone: Timezone;
+}
+
+const initialState: IPreferencesState = {
+  theme: Theme.LIGHT,
```

Comment:
Does it make more sense for the default to be system?

Just thinking I would rather things snapped to my system state by default if I want most things to be e.g. dark, that is my way to avoid having to set it everywhere.

Reply:
but i see what you're saying, i guess not defaulting to 'system' kind of defeats the purpose of having 'system' as does prevent you from having to go in and change the settings in each app. I was kind of using GitHub as a basis though as you have to 'opt in' to 'System'
![CleanShot 2023-06-05 at 11 04 05](https://github.com/flipt-io/flipt/assets/209477/b018e704-0824-4405-bad3-96bbb9b8d09a)